lib/msal-common/test/cache/CacheManager.spec.ts

```diff
@@ -239,6 +245,29 @@ describe("CacheManager.ts test cases", () => {
         const filterTwo: AccountFilter = { homeAccountId: "Wrong Id" };
         accounts = cacheManager.getAccountsFilteredBy(filterTwo);
         expect(Object.keys(accounts).length).to.eql(0);
+
+        sinon.stub(TrustedAuthority, "getCloudDiscoveryMetadata").callsFake(() => {
+            return {
+                preferred_cache: Constants.DEFAULT_AUTHORITY_HOST, 
+                preferred_network: Constants.DEFAULT_AUTHORITY_HOST, 
+                aliases: [Constants.DEFAULT_AUTHORITY_HOST]}
+        });
+        const filterThree: AccountFilter = { environment: "login.microsoftonline.com" };
```

Comment:
Can we break each of these tests into a separate test and make the current `it` a `describe`?

Reply:
done.